### PR TITLE
Update Travis CI Scripts

### PR DESCRIPTION
**Pull request by vchimev**
Fix publish to S3.

### CODE REVIEW

**Comment by vchimev** [comment]
run ci

**Comment by vchimev** [comment]
run ci

File: .travis/add-publishConfig.js [hunk 1]
```diff
@@ -1,25 +1,22 @@
 #!/usr/bin/env node
 
-var fsModule = require("fs");
+const tag = "next";
```
Comment (rosen-vladimirov) [inline .travis/add-publishConfig.js]: this variable is not used anywhere

Comment (vchimev) [inline .travis/add-publishConfig.js]: Thanks! :)

**Pull request merged**